src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -55,8 +55,8 @@ const TOS_ACK_LOCAL_STORAGE_KEY = 'tos-ack';
  * An access key to be displayed
  * @typedef {Object} ServerListEntry
  * @prop {string} id
+ * @prop {string|null} accountId
```

Comment:
👍 

This will make it very easy to support multiple accounts of the same cloud provider in the future!